cli/README.md

```diff
@@ -1,55 +1,74 @@
-# Xata CLI
+# Command Line Interface (CLI)
 
-## Installing
+Xata provides a CLI as a convenience for developers. It is intended to provide feature parity with Xata's [Web UI](https://docs.xata.io/getting-started) (User Interface) to enable developers to build applications at **rapid pace** with **as little friction as possible**.
 
-The recommended way of installing the CLI is by installing it as an npm package globally:
+## Installation
 
-```bash
-npm install @xata.io/cli -g
+Our CLI is distributed both as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package** for developers who work with Node.js, and a **cross-platform binary** for those who don't. Below, you'll find commands that you can copy and paste into your platform's shell that will help you install the CLI on your specific platform.
+
+![Installing the Xata CLI](/screenshots/CliInstall.gif)
+
+### Installing with npm
+
+To install the Xata CLI globally, run the following command:
+
+```sh
+npm i --location global @xata.io/cli
 ```
 
-Now you can just run `xata --help` to see all the available commands.
+This will install the Xata CLI. We recommend installing it globally because it becomes much more convenient to work with, having you invoke a `xata` command instead of the more convoluted alternative with a project-scoped installation (`npm run xata`), or an npx-based installation (`npx @xata.io/cli`). The alternative approaches also work depending on your preference.
+
+### Installing the Binary without npm
+
+Our CLI can also be installed as a cross-platform binary for our users who do not have `npm` installed on their systems. Depending on your platform of choice, here are installation instructions to get you started.
 
-It's perfectly fine to install it locally in your project if you wish:
+#### macOS and Linux
 
-```bash
-npm install @xata.io/cli --save-dev
+Paste the following command into your terminal and hit Enter.
+
+```sh
+curl -L  https://xata.io/install.sh | sh
 ```
 
-## Authentication, API keys and projects
+This command fetches the latest CLI install script and runs it in your shell. Piping a script to bash can raise eyebrows, so the code for this is open and can be visited in your browser in case you find your eyebrows raised.
+
+#### Windows
+
+Paste the following command into your terminal and hit enter.
 
-Almost all commands can work in two different modes: globally or scoped to a project. You can configure a project with `xata init`. This will create a `.xatarc` file (although `.xatarc.json` and `package.json` are also supported) and an `.env` file.
```

Comment:
I think explaining the difference between the two modes (global vs project) is important.

Reply:
Done in https://github.com/xataio/client-ts/pull/400/commits/9778eeba7b810f0c4ca53f9fdafee2af990bce9d